src/panels/lovelace/cards/hui-vertical-stack-card.ts

```diff
@@ -0,0 +1,95 @@
+import { html, LitElement } from "@polymer/lit-element";
+
+import computeCardSize from "../common/compute-card-size.js";
+import createCardElement from "../common/create-card-element.js";
+
+import { LovelaceCard, LovelaceConfig, LovelaceCardElement } from "../types";
+import { HomeAssistant } from "../../../types";
+
+interface Config extends LovelaceConfig {
+  cards: LovelaceConfig[];
+}
+
+class HuiVerticalStackCard extends LitElement implements LovelaceCard {
+  protected config: Config;
+  private _hass: HomeAssistant;
+
+  static get properties() {
+    return {
+      config: {},
+    };
+  }
+
+  set hass(hass: HomeAssistant) {
+    this._hass = hass;
+    for (const el of this.shadowRoot!.querySelectorAll("#root > *")) {
+      const element = el as LovelaceCardElement;
+      element.hass = this._hass;
+    }
+  }
+
+  public getCardSize() {
+    let totalSize = 0;
+    for (const element of this.shadowRoot!.querySelectorAll("#root > *")) {
```

Comment:
this won't work as we haven't rendered yet here. Not sure if Lit has a synchronous render function that we can use.